packages/babel-plugin-transform-runtime/src/runtime-corejs3-definitions.js

```diff
@@ -185,47 +185,51 @@ export default () => {
+      bind: { stable: true, path: "bind", types: ["function"] },
+      codePointAt: { stable: true, path: "code-point-at", types: ["string"] },
+      codePoints: { stable: false, path: "code-points", types: ["string"] },
       concat: { stable: true, path: "concat", types: ["array"] },
-      copyWithin: { stable: true, path: "copy-within" },
-      endsWith: { stable: true, path: "ends-with" },
-      entries: { stable: true, path: "entries" },
-      every: { stable: true, path: "every" },
-      fill: { stable: true, path: "fill" },
-      filter: { stable: true, path: "filter" },
-      find: { stable: true, path: "find" },
-      findIndex: { stable: true, path: "find-index" },
-      flags: { stable: true, path: "flags" },
-      flatMap: { stable: true, path: "flat-map" },
-      flat: { stable: true, path: "flat" },
-      forEach: { stable: true, path: "for-each" },
-      includes: { stable: true, path: "includes" },
-      indexOf: { stable: true, path: "index-of" },
-      keys: { stable: true, path: "keys" },
-      lastIndexOf: { stable: true, path: "last-index-of" },
-      map: { stable: true, path: "map" },
-      matchAll: { stable: false, path: "match-all" },
-      padEnd: { stable: true, path: "pad-end" },
-      padStart: { stable: true, path: "pad-start" },
-      reduce: { stable: true, path: "reduce" },
-      reduceRight: { stable: true, path: "reduce-right" },
-      repeat: { stable: true, path: "repeat" },
-      replaceAll: { stable: false, path: "replace-all" },
-      reverse: { stable: true, path: "reverse" },
-      slice: { stable: true, path: "slice" },
-      some: { stable: true, path: "some" },
-      sort: { stable: true, path: "sort" },
-      splice: { stable: true, path: "splice" },
-      startsWith: { stable: true, path: "starts-with" },
-      trim: { stable: true, path: "trim" },
-      trimEnd: { stable: true, path: "trim-end" },
-      trimLeft: { stable: true, path: "trim-left" },
-      trimRight: { stable: true, path: "trim-right" },
-      trimStart: { stable: true, path: "trim-start" },
-      values: { stable: true, path: "values" },
+      copyWithin: { stable: true, path: "copy-within", types: ["array"] },
+      endsWith: { stable: true, path: "ends-with", types: ["string"] },
+      entries: { stable: true, path: "entries", types: ["array"] },
+      every: { stable: true, path: "every", types: ["array"] },
+      fill: { stable: true, path: "fill", types: ["array"] },
+      filter: { stable: true, path: "filter", types: ["array"] },
+      find: { stable: true, path: "find", types: ["array"] },
+      findIndex: { stable: true, path: "find-index", types: ["array"] },
+      flags: { stable: true, path: "flags", types: ["regexp"] },
+      flatMap: { stable: true, path: "flat-map", types: ["array"] },
+      flat: { stable: true, path: "flat", types: ["array"] },
+      forEach: { stable: true, path: "for-each", types: ["array"] },
```

Comment:
This is also for map/set, but I _think_ that Babel handles them as "unknown" and thus adds the polyfill anyway?

Reply:
> I think that Babel handles them as "unknown" and thus adds the polyfill anyway?

Correct.

Reply:
Could you add a test? I don't want to introduce bugs in case we decide to optimize the logic also for map/set other than array/etc.

Reply:
The problem with `Map` and `Set` that even if we will add naive detection of them to `typeAnnotationToString` to prevent adding unnecessary `.forEach` polyfill, `Map` anyway could be transpiled to something like `_Map` by `runtime` transform.